docs/sdg/sdg-flow-yaml.md

```diff
@@ -0,0 +1,157 @@
+# SDG API :: Add a file format for defining custom Flows
```

Comment:
Was the double colon intentional?

Reply:
I like double colon! Changed it for you :heart:

Reply:
I'm honored, thank you Sir!